app/src/main/res/layout-land/fragment_share_payment_link.xml

```diff
@@ -0,0 +1,120 @@
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toEndOf="@id/mid_guideline"
+      app:layout_constraintTop_toTopOf="parent"
+      app:layout_constraintVertical_chainStyle="packed"
+      style="@style/ShareLinkTextStyle.Title.Landscape"
+      />
+
+  <EditText
+      android:id="@+id/note"
+      android:layout_width="0dp"
+      android:layout_height="40dp"
+      android:layout_marginEnd="@dimen/big_margin"
+      android:layout_marginStart="@dimen/big_margin"
+      android:layout_marginTop="@dimen/big_margin"
+      android:background="@null"
+      android:gravity="start|center_vertical"
+      android:hint="@string/askafriend_share_add_message_body"
+      android:maxLength="50"
+      android:maxLines="2"
+      android:textColorHint="@color/grey_alpha_6b"
+      app:layout_constraintBottom_toBottomOf="@id/bottom_separator"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toEndOf="@id/mid_guideline"
+      app:layout_constraintTop_toBottomOf="@id/share_link_title"
+      app:layout_constraintVertical_chainStyle="packed"
+      style="@style/ShareLinkTextStyle"
+      />
+
+  <View
+      android:id="@+id/bottom_separator"
+      android:layout_width="match_parent"
+      android:layout_height="1dp"
+      android:layout_marginTop="252dp"
+      android:background="@color/layout_separator_color"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent"
+      />
+
+  <Button
+      android:id="@+id/share_btn"
+      android:layout_width="wrap_content"
+      android:layout_height="36dp"
+      android:layout_marginBottom="@dimen/normal_margin"
+      android:layout_marginEnd="@dimen/big_margin"
+      android:layout_marginTop="@dimen/normal_margin"
+      android:drawablePadding="8dp"
+      android:drawableStart="@drawable/ic_share"
+      android:text="@string/askafriend_share_button"
+      app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintHorizontal_bias="1"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toBottomOf="@id/bottom_separator"
+      style="@style/ButtonStyle"
+      />
+
+  <Button
+      android:id="@+id/close_btn"
+      android:layout_width="wrap_content"
+      android:layout_height="36dp"
+      android:layout_marginBottom="@dimen/normal_margin"
+      android:layout_marginEnd="@dimen/big_margin"
+      android:layout_marginTop="@dimen/normal_margin"
+      android:text="@string/close"
+      app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintEnd_toStartOf="@id/share_btn"
+      app:layout_constraintHorizontal_bias="1"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toBottomOf="@id/bottom_separator"
+      style="@style/CancelButtonStyle"
+      />
```

Comment:
The colour of the cancel button is not the one specified on zeplin, should we change the style to match it in this ticket, in another one or just leave it like it is right now?